bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/handler/RotelHandler.java

```diff
@@ -1696,9 +1696,11 @@ public void onNewMessageEvent(EventObject event) {
                 case KEY_PCUSB_CLASS:
                     logger.debug("PC-USB Audio Class is set to {}", value);
                     break;
+                case KEY_PRODUCT_TYPE:
```

Comment:
What is the difference between `KEY_PRODUCT_TYPE` and `KEY_MODEL` (as well as `KEY_PRODUCT_VERSION` vs. `KEY_VERSION`)? Do they belong to different protocols so they cannot both be received, which would then conflict?

Reply:
Yes, two different ASCII protocols.